### DISCUSSION TITLE
dpgen init_bulk error: find no valid outcar in sys sys-0016, check if your vasp md simulation is correctly done

### DISCUSSION THREAD

**Discussion by sunlight905** [deepmodeling/dpgen#1425] [Q&A]
When I do aluminum test using dpgen init_bulk.
but after running dpgen init_bulk, there are some errors.
Traceback (most recent call last):
  File "/home/sunlight/anaconda3/envs/dpgen0.12.0/bin/dpgen", line 10, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/sunlight/.local/lib/python3.11/site-packages/dpgen/main.py", line 233, in main
    args.func(args)
  File "/home/sunlight/.local/lib/python3.11/site-packages/dpgen/data/gen.py", line 1596, in gen_init_bulk
    coll_vasp_md(jdata)
  File "/home/sunlight/.local/lib/python3.11/site-packages/dpgen/data/gen.py", line 1097, in coll_vasp_md
    raise RuntimeError(
RuntimeError: MD dir: /home/sunlight/work2/dpgen/jisuan/al/init/bcc/albcc.01x01x01/02.md: find no valid outcar in sys sys-0016, check if your vasp md simulation is correctly done.

The vasp outcar is finished, but can not obtain the potential.
The main problem is in all the 000000 folder, the calcualtion in the 000000 folder can not used for obtained the potential, and show the error "02.md: find no valid outcar in sys sys-0016, check if your vasp md simulation is correctly done."
and the outcar of the other folder such as 000001,000002 can be correctly used for the potential.
[dpgen.alinit.zip](https://github.com/deepmodeling/dpgen/files/13645250/dpgen.alinit.zip)

Thank you!

**Comment by njzjz** [comment]
The file you uploaded doesn't contain an outcar file.

**Reply by sunlight905** [reply]
Thanks for your reply!
This is the outcar for aluminum. 
[albcc.01x01x01.zip](https://github.com/deepmodeling/dpgen/files/13655133/albcc.01x01x01.zip)
[scale-0.720.zip](https://github.com/deepmodeling/dpgen/files/13655135/scale-0.720.zip)

On the other, for scale-0.720 000000, the vasp can not finish rightly.
and show the error,
Information: wavefunction orthogonal band   40  0.7680
Information: wavefunction orthogonal band   40  0.7139
Information: wavefunction orthogonal band   40  0.7999
Information: wavefunction orthogonal band   40  0.2486
 -----------------------------------------------------------------------------
|                                                                             |   
|     EEEEEEE  RRRRRR   RRRRRR   OOOOOOO  RRRRRR      ###     ###     ###     |   
|     E        R     R  R     R  O     O  R     R     ###     ###     ###     |   
|     E        R     R  R     R  O     O  R     R     ###     ###     ###     |   
|     EEEEE    RRRRRR   RRRRRR   O     O  RRRRRR       #       #       #      |   
|     E        R   R    R   R    O     O  R   R                               |   
|     E        R    R   R    R   O     O  R    R      ###     ###     ###     |   
|     EEEEEEE  R     R  R     R  OOOOOOO  R     R     ###     ###     ###     |   
|                                                                             |   
|     ERROR in subspace rotation PDSYEVX/ PZHEEVX: I2,IFAIL= 1 2              |   
|                                                                             |   
|       ---->  I REFUSE TO CONTINUE WITH THIS SICK JOB ... BYE!!! <----       |   
|                                                                             |   
 -----------------------------------------------------------------------------

1
1
1
1
1
Thank you!
